CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contribution Guide #
+# Contribution Guide
```

Comment:
Yes, both are valid. I mean it's better to use the consistent markdown style in this document, and `# header` is more common and shorter :)